### ISSUE TITLE
PALADIN/RET Never recommends Divine Toll

### ISSUE THREAD

**Issue by Fusroedah** [Hekili/hekili#2363]
### Before You Begin

- [X] I confirm that I have downloaded the latest version of the addon.
- [X] I am not playing on a private server.
- [X] I checked for an [existing, open ticket](https://github.com/Hekili/hekili/labels/recommendation) for this issue and was not able to find one.
- [X] I edited the title of this issue (above) so that it describes the issue I am reporting.
- [X] I am reporting an issue with the default priority included with the specialization (imported or edited priorities are not supported).

### Describe the Issue

Hekili is never recommending Divine Toll in either ST or AOE situations for Retribution. I have confirmed that I have the talent and confirmed it is not a toggle within the Retribution settings.

### How to Reproduce

1. Enter game as Retribution Paladin
2. Select a talent spec that includes the Divine Toll talent
3. Engage with both ST and AOE dummies for a minute and Divine Toll is never a recommended ability

### Snapshot (Link)

https://pastebin.com/TCi0cGXx

### Raidbots Sim Report (Link)

https://www.raidbots.com/simbot/report/kpf2VKBWibQPVk1UE4X5cb

**Comment by Hekili** [comment]
In this case, it's not that Divine Toll is being impacted by a toggle directly, but it is impacted by other abilities being on a toggle.

<details>
<summary>31.  divine_toll ( generators - 5 )</summary>

```
- this entry's criteria FAILS: holy_power.current[0.00] <= 2 & ! debuff.judgment.up[false] & ( ! talent.seraphim.enabled[true] | buff.seraphim.up[false] | cooldown.seraphim.remains[0.00] > 15 ) & ( ! raid_event.adds.exists[false] | raid_event.adds.in[3600.00] > 30 | raid_event.adds.up[false] ) & ! talent.final_reckoning.enabled[true] & ( ! talent.execution_sentence.enabled[false] | fight_remains[30.00] < 8 | spell_targets.divine_storm[1.00] >= 5 ) & ( cooldown.avenging_wrath.remains[0.00] > 15 | cooldown.crusade.remains[0.00] > 15 | boss[false] & fight_remains[30.00] < 8 )
```

</details>

The above entry is never going to pass because you have Final Reckoning talented -- that entry is to be used when you haven't talented for FR.

The other entry for Divine Toll is in the `es_fr_active` action list, which is invoked when either Execution Sentence is active or Final Reckoning is active.

```
25.  call_action_list:es_fr_active ( default - 4 )
Time-sensitive Criteria FAIL at +1.22 with no valid rechecks - debuff.execution_sentence.up[false] | debuff.final_reckoning.up[false]
```

Because neither debuff is active, the condition fails.  Of course, you don't have Execution Sentence talented, so the question is really re: Final Reckoning.  When would Final Reckoning get used?  It's in the `cooldowns` list, with a bunch of criteria, none of which matter because you have Final Reckoning disabled.

```
23.  final_reckoning ( cooldowns - 14 ) - ability disabled ( toggle a )
```

tl;dr:  If you had your Cooldowns toggle active, the addon would have the option to recommend Final Reckoning, which also is generally synchronized with Seraphim since you've talented for that, which would ultimately also enable Divine Toll to be cast.

**Comment by Hekili** [comment]
I'm adding a spec option to desync Divine Toll if abilities are disabled that might help your situation.
